driver/src/contracts/stablex_contract.rs

```diff
@@ -61,7 +61,13 @@ impl<'a> StableXContractImpl<'a> {
 
 #[cfg_attr(test, automock)]
 pub trait StableXContract {
-    fn get_current_auction_index(&self) -> Result<U256>;
+    /// Retrieve the index for the auction that is currently accepting orders.
+    fn get_current_auction_index(&self) -> Result<u32>;
+
+    /// Retrieve the amount of time remaining in the current batch in
+    /// milliseconds.
+    fn get_current_auction_remaining_seconds(&self) -> Result<U256>;
```

Comment:
Comment says milliseconds, function says seconds. Should return Duration?

Reply:
I see that we return Duration from the orderbook. So its fine to just fix the comment.

Reply:
Nice catch!